sdk/spring/spring-cloud-azure-starter-active-directory-b2c/pom.xml

```diff
@@ -171,18 +171,20 @@
             <phase>prepare-package</phase>
             <configuration>
               <target>
-                <echo>Deleting existing ${project.basedir}/javadocTemp</echo>
+                <echo>Deleting existing ${project.basedir}/javadocTemp and
+                  ${project.basedir}/sourceTemp</echo>
                 <delete includeEmptyDirs="true" quiet="true">
                   <fileset dir="${project.basedir}/javadocTemp"/>
+                  <fileset dir="${project.basedir}/sourceTemp"/>
                 </delete>
                 <echo>Copying ${project.basedir}/README.md to
                   ${project.basedir}/javadocTemp/README.md
                 </echo>
-                <copy file="${project.basedir}/../README.md" tofile="${project.basedir}/javadocTemp/README.md"/>
+                <copy file="${project.basedir}/../README.md" tofile="${project.basedir}/javadocTemp/README.md" failonerror="false"/>
```

Comment:
Maybe we should update the sparse checkout instead of setting `failonerror="false"` here.

Setting `failonerror="false"` may cause the released javadoc / sourcejar does not contains README.md.

We should avoid uncertain behavior.

Reply:
I think it's okay, the sdk/spring/REAME.md will always be there, and what we want to achieve here is to avoid exceptions when sparse checkout does not include the sdk/spring/README.md file.

Reply:
What we also want to avoid is adding a README.md in each starter's folder.

Reply:
And even if we ship some Javadoc and source jar files without the README, I think it'll still be ok. The starters don't have any source code or Javadoc.

Reply:
The key point is `We should avoid uncertain behavior`.

```
Customer: Does the Java doc and source jar contains README?
We: We are not sure.
```
That's not professional.